daml-foundations/daml-ghc/src/DA/Service/Daml/Compiler/Impl/Dar.hs

```diff
@@ -47,26 +48,34 @@ buildDar ::
   -> FilePath
   -> [(T.Text, BS.ByteString)]
   -> [FilePath]
+  -> [(String, BS.ByteString)]
   -> String
   -> IO BS.ByteString
-buildDar dalf modRoot dalfDependencies fileDependencies name = do
+buildDar dalf modRoot dalfDependencies fileDependencies dataFiles name = do
```

Comment:
Could you add a comment here explaining what `dataFiles` are used for?

Reply:
will do.